### DISCUSSION TITLE
Lazygit view

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#4698] [General]
How do I increase the lazygit window triggered by `<leader>gg` - both the height and width?

**Comment by dpetka2001** [comment]
```lua
map("n", "<leader>gg", function()
  LazyVim.lazygit({ size = { width = 1.0, height = 1.0 }, esc_esc = true, cwd = LazyVim.root.get() })
end, { desc = "Lazygit (root dir)" })
map("n", "<leader>gG", function()
  LazyVim.lazygit({ size = { width = 1.0, height = 1.0 }, esc_esc = true })
end, { desc = "Lazygit (cwd dir)" })
map("n", "<leader>gf", function()
  local git_path = vim.api.nvim_buf_get_name(0)
  LazyVim.lazygit({ args = { "-f", vim.trim(git_path) }, size = { width = 1.0, height = 1.0 }, esc_esc = true })
end, { desc = "Lazygit current file history" })
```
Where `local map = vim.keymap.set` for me and I also included `esc_esc = true` for personal preference, but you can totally skip that if you like the default behavior.

**Reply by Raagh** [reply]
and how do you get this lazyvim module now that we use snacks?

I used to do this but now it complains about not using snacks and seems it is not full screen anymore even with this setting

```lua
local Util = require("lazyvim.util")
set_keymap("n", "<leader>gg", function()
  Util.lazygit({ cwd = Util.root.git(), size = {
    width = 1,
    height = 1,
  }, border = "none" })
end, { desc = "Lazygit" })
```

**Reply by Raagh** [reply]
![image](https://github.com/user-attachments/assets/8698435d-2395-4595-904c-af312b3dbccb)

**Reply by dpetka2001** [reply]
```lua
{
  "folke/snacks.nvim",
  opts = {
      lazygit = {
        win = {
          height = 0,
          width = 0,
        },
      },
  },
}
```

**Reply by Raagh** [reply]
that is awesome that snacks handles this config now. thanks

**Comment by folke** [comment]
You'll soon be able to configure default options for floats and terminals so you no longer need to change the keymaps.

Stay tuned :)
